api-reference/10 UI Components/dxMap/1 Configuration/providerConfig/mapId.md

```diff
@@ -5,7 +5,7 @@ default: ''
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a [map ID](https://developers.google.com/maps/documentation/get-map-id) for the `google` [provider](/Documentation/ApiReference/UI_Components/dxMap/Configuration/#provider).
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This property is required if you use [advanced markers](/Documentation/ApiReference/UI_Components/dxMap/Configuration/providerConfig/#useAdvancedMarkers).
```

Comment:
We've replaced old markers with advanced mostly to avoid console warning about deprecation from google, there are no advantages from new Advanced markers for users. Thus I beleive we shouldn't focus attention that now markers are advanced if possible. 

May be just `This property is required if you use markers`